libraries/AP_Math/rotations.h

```diff
@@ -63,10 +63,12 @@ enum Rotation : uint8_t {
     ROTATION_ROLL_270_PITCH_270  = 35,
     ROTATION_ROLL_90_PITCH_180_YAW_90 = 36,
```

Comment:
This one is the same as ROLL_270_YAW_270, admittedly that is not one that we have, but is is much easier to understate as two numbers rather than 3.